PrivateCollections/Germany/Neukirchen005.xml

```diff
@@ -301,9 +460,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         reused leaves with text, originating from a different, more recent manuscript.</decoNote>
                           <decoNote xml:id="b5" type="bindingMaterial">
                           <material key="wood"/>
-                          <material key="leather"></material>
-                          <material key="textile"></material>
+                          <material key="leather"/>
+                          <material key="textile"/>
                           </decoNote>
+                          <decoNote xml:id="b6" type="Other">
+                            <term key="leafStringMark"></term>
+                            Leaf string markers (coloured or uncoloured threads) are
```

Comment:
Should be
` <term key="leafStringMark">Leaf string markers</term> (coloured...)`

Reply:
Done.